variables.yaml.example

```diff
@@ -28,8 +32,13 @@ runner_fully_qualified_domain_name: ''
 
 requested_cpu_per_pod: '0.7'
 requested_memory_per_pod: '2048Mi'
+
+# How many instances of locust to run
```

Comment:
Locust should not be mentioned as that's an implementation detail of the test runner. Something like `number_of_pods` or `number_of_load_injectors` woulds be better.

Reply:
I've updated the text to remove reference to locust, but still keep it explanatory